code/script.js

```diff
@@ -26,22 +42,221 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+  // Scroll to last message when there are too many in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// The dialogue starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hi there! What's your name?`, 'bot')
+}
+
+const moodQuestion = () => {
```

Comment:
Suggestion to remove the start button after being used so that you cant press it when the bot has started. This also applies for the text input when you haven't pressed the start button.

Reply:
Right! Important UX details.